statistics/handle/handle.go

```diff
@@ -298,6 +298,7 @@ func (h *Handle) Update(is infoschema.InfoSchema, opts ...TableStatsOpt) error {
 		tbl.Name = getFullTableName(is, tableInfo)
 		tbl.TblInfoUpdateTS = tableInfo.UpdateTS
 		tables = append(tables, tbl)
+		oldCache.FreshTableCost(tbl.PhysicalID)
```

Comment:
tables are not put into oldCache yet, oldCache is not yet changed here.

Reply:
updated.